v2/tools/transformer/src/transformations.js

```diff
@@ -164,25 +164,8 @@ function transformOneOf(parentObject, api) {
   delete parentObject.oneOf;
 }
 
-// Temporary transformation for versioned api
-function applyResponseTransformation(doc){
-  const hasResponses = doc.components && doc.components.responses 
-  if (hasResponses) {
-    for (const responseKey of Object.keys(doc.components.responses)) {
-      response = doc.components.responses[responseKey];
-      if (response.content && response.content["application/json"]) {
-        response.content["application/vnd.atlas.unversioned+json"] =
```

Comment:
Removed transformation. 
This transformation did not work properly as the versioned API only supports singular content type. We could only use custom template.